lib/util.cpp

```diff
@@ -542,3 +542,38 @@ bool process_exists(int pid) {
 }
 
 #endif
+
+unsigned int random_int() {
+    unsigned int n;
+#if defined(_WIN32)
+#if defined(__CYGWIN32__)
+    HMODULE hLib=LoadLibrary((const char *)"ADVAPI32.DLL");
+#else
+    HMODULE hLib=LoadLibraryA("ADVAPI32.DLL");
+#endif
+    if (!hLib) {
+        fprintf(stderr, "Error: can't load ADVAPI32.DLL\n");
+    }
+    BOOLEAN (APIENTRY *pfn)(void*, ULONG) =
```

Comment:
If the library was not loaded correctly two lines above - this will fail with an exception